### PR TITLE
Add schema version with filter property

### PR DESCRIPTION
**Pull request by ArtyomyuS**
Issue: https://github.com/datacontract/datacontract-specification/issues/67

### CODE REVIEW

**Comment by ArtyomyuS** [comment]
Hey @jochenchrist , is there more related to schema definition to be done here? Thank you in advance

**Comment by jochenchrist** [comment]
Update README.
Also, I would introduce a `description` field to explain the filter.

**Comment by ArtyomyuS** [comment]
Strange I've added description for both filter and expression, is it another description you expect?
```
 "filter": {
            "description": "Specifies any filters to apply to the data.",
            "type": "object",
            "properties": {
              "expression": {
                "type": "string",
                "description": "The filter expression to apply to the model.",
                "examples": [
                    "order_id > 10 AND order_id < 20",
                    "order_id = '1000'",
                    "country_code in ('DE', 'AT', NULL)",
                    "country_code ILIKE '%D%'"
                ]
              }
            }
          },
```

**Comment by ArtyomyuS** [comment]
Updated readme, applied also a formatting. Let me know 👍

**Comment by ArtyomyuS** [comment]
Hey @jochenchrist, any update on this one? I would like to start over with datactonract-cli to produce SQL based on filter expressions :)

File: README.md [hunk 1]
```diff
@@ -8,28 +8,38 @@
...
 
-A _data contract_ is a document that defines the structure, format, semantics, quality, and terms of use for exchanging data between a data provider and their consumers. 
-Think of an API, but for data. 
-A data contract is implemented by a data product or other data technologies, even legacy data warehouses. 
-Data contracts can also be used for the input port to specify the expectations of data dependencies and verify given guarantees.
-
-The _data contract specification_ defines a YAML format to describe attributes of provided data sets. 
-It is data platform neutral and can be used with any data platform, such as AWS S3, Google BigQuery, Azure, Databricks, and Snowflake. 
-The data contract specification is an open initiative to define a common data contract format. 
+A _data contract_ is a document that defines the structure, format, semantics, quality, and terms of use for exchanging
```
Comment (jochenchrist) [inline README.md:11]: Too many changes in the PR (auto format?). Please keep the PR focused on the functional change.

**Review by jochenchrist — CHANGES_REQUESTED** [review]

**Comment by jochenchrist** [comment]
> Strange I've added description for both filter and expression, is it another description you expect?
> 
> ```
>  "filter": {
>             "description": "Specifies any filters to apply to the data.",
>             "type": "object",
>             "properties": {
>               "expression": {
>                 "type": "string",
>                 "description": "The filter expression to apply to the model.",
>                 "examples": [
>                     "order_id > 10 AND order_id < 20",
>                     "order_id = '1000'",
>                     "country_code in ('DE', 'AT', NULL)",
>                     "country_code ILIKE '%D%'"
>                 ]
>               }
>             }
>           },
> ```

I mean a new property next to `expression`

File: README.md (hunk 1 above)
Comment (ArtyomyuS) [reply README.md]: Done roll-back on auto-complete.

**Comment by ArtyomyuS** [comment]
Hey @jochenchrist , sorry to bother, any new updates on this one? Thanks

**Comment by simonharrer** [comment]
@ArtyomyuS let's continue the discussion at https://github.com/datacontract/datacontract-specification/issues/67 (see my last comment there).

**Comment by jochenchrist** [comment]
As discussed in https://github.com/datacontract/datacontract-specification/issues/67 there are better options: lineage, quality, and config options.
Closing the issue as won't fix, but nevertheless thanks for contributing!

**Pull request closed without merging**